inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/conf/TriggerProfile.java

```diff
@@ -20,12 +20,22 @@
 import org.apache.inlong.agent.constant.JobConstants;
 import org.apache.inlong.agent.pojo.JobProfileDto;
 import org.apache.inlong.common.pojo.agent.DataConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
 
 /**
  * profile used in trigger. Trigger profile is a special job profile
  */
 public class TriggerProfile extends JobProfile {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(TriggerProfile.class);
+
+    private final SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
SimpleDateFormat is not thread-safe

Reply:
OK, I replaced `SimpleDateFormat ` by `DateTimeFormatter`.